ccm_web/package.json

```diff
@@ -21,6 +21,8 @@
     "ltijs": "~5.7.3",
     "ltijs-sequelize": "~2.4.2",
     "notistack": "^1.0.5",
+    "papaparse": "^5.3.0",
+    "prop-types": "^15.7.2",
```

Comment:
Can we remove `prop-types` now? MIght be a sub-dependency of something else, but if we don't import it directly, probably better to remove it. What is `papaparse`?

Reply:
I can check on removing prop-types.

Papaparse is the csv parsing library that was used by old ccm so I continued to use it.

Reply:
Hm, seems widely used. May not be as popular or current as https://www.npmjs.com/package/csv-parse It looks like it's written in TypeScript and has had a more recent release. Probably fine either way, I'll let you decide.

So are you looking at the old FE as you work on this? Nice strategy, though I wouldn't necessarily assume all those patterns/choices are still the preferred ones.

Reply:
I referred to the old ccm to see what logic was  employed b/c it seemed like the  definitive source of truth to me.  I did choose that parsing library because it was used there, but the code is new.

Maybe there is an advantage to using a newer library/one written in typescript however papaparse does have @types support and it's working now.

It would  probably be only a minor task to switch out libraries, but it wouldn't be a straight drop in.  I would not chose to change libraries at this time, since as far as I know it's working and there are concerns about being on schedule...

Reply:
I would not necessarily change the library now. it's not worth it. Since we have more csv file upload features like `add sections`, `3rd-party gradebook` , 'Bulk user add` we  could try this library  npmjs.com/package/csv-parse if Chris wanted to.